CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.17.0 (2023-08-02)
+
+### New features
+
+- Add `pika::wait` to wait for the runtime to be idle. ([#704](https://github.com/pika-org/pika/pull/704))
+- The debugger can now be attached after printing failure information. Previously the debugger could be attached before printing the error information. ([#712](https://github.com/pika-org/pika/pull/712))
```

Comment:
I would word this the converse: Failure information is now printed before attaching a debugger. I know the difference might sound trivial, but at least to me that puts the focus on the failure information rather than attaching the debugger.